vite-project/src/components/Modal/AddLinkModal.jsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import "../../style/AddLinkModal.css";
+
+function AddLinkModal({ onClose, cardListMenuData }) {
+  const [selectedFolder, setSelectedFolder] = useState("");
+
+  const handleFolderSelect = folderName => {
+    setSelectedFolder(folderName);
+  };
+
+  return (
+    <div className="addlinkmodal-section">
+      <div className="addlinkmodal-title-section">
+        <div className="addlinkmodal-maintitle">폴더에 추가</div>
+        <div className="addlinkmodal-subtitle">링크 주소</div>
+      </div>
+      <div className="addlinkmodal-folder-list">
+        {cardListMenuData?.data?.map(folder => (
```

Comment:
이름으로 봤을 때, 혼동을 줄 수 있어서 제안드려요
```suggestion
        {folders?.data?.map(folder => (
```